etc/db-config.yaml

```diff
@@ -235,12 +239,16 @@
             type: string
             default_value: ""
             public: true
-            description: Team category for users that register themselves with the system. Self-registration is disabled if this field is left empty.
+            description: Team category for users that register themselves with the system
         -   name: data_source
             type: int
             default_value: 0
             public: false
-            description: "Source of data. Choices: 0 = all local, 1 = configuration data external, 2 = configuration and live data external"
+            description: Source of data
```

Comment:
Although not directly related to this change: I think this needs more clarification. Maybe something like:
`Source of data: used to indicate whether internal or external IDs are exposed in the API. Option 1 is typically used when loading configuration data from the ICPC CMS, and option 2 when running DOMjudge as "shadow system".`